### ISSUE TITLE
HighContrast icon theme

### ISSUE THREAD

**Issue by Obsidien** [lwindolf/liferea#148]
GNOME[1] and Gtk (integrated in 3.15.5) support a second icons’ theme, HighContrast. It would be great for accessibility that Liferea offers such an icon.

[1] https://wiki.gnome.org/Initiatives/GnomeGoals/HighContrastAppIcons

**Comment by lwindolf** [comment]
I'm personally just not able to provide such an icon. I posted a help request in the development blog though: http://lzone.de/Help-Needed-High-Contrast-Icon.

@Obsidien What about you, would you consider contributing an icon?

**Comment by Obsidien** [comment]
[3.15.3, sorry… well, the devel version]

The simpler way is probably to copy an RSS icon in black & white following the guidelines (big lines and white border). Not so difficult to do, I might (!) have a look after GNOME code freeze (next Monday).

**Comment by stevanov** [comment]
Gave it a quick shot in GIMP just now. 
![Example High Contrast Logo](http://stevenkramer.nl/lifereahc.png)
I think it works pretty well in high contrast.

**Comment by Obsidien** [comment]
Just tested, lines are too thin for a good readability / accessibility, that are the goals of this theme. The earth motif should probably disappear if you go in that direction, and “NEWS” should probably take all the length of the icon (and I’m not sure it would be big enough, but I didn’t try). But I think a more “simple” symbol should be chosen, like a big RSS icon.

**Comment by stevanov** [comment]
I'll be honest in that i did not bother trying it myself... Just tried and you're right, there isn't a lot left too see when you resize it to 48x48. The alternative is a bit boring but it is universally recognizable. I tried to keep the lines on this one a bit more like other High Contrast icons. 
![RSS](https://dl.dropboxusercontent.com/u/3879064/lifereahc2.png)
I also attempted to incorporate the newspaper in the background but it wasn't very clear that it was a paper anymore. 
By the way, maybe the default High Contrast pack already comes with an RSS icon which we could use?
